src/couch_httpd_auth.erl

```diff
@@ -366,6 +366,18 @@ handle_session_req(#httpd{method='DELETE'}=Req, _AuthModule) ->
 handle_session_req(Req, _AuthModule) ->
     send_method_not_allowed(Req, "GET,HEAD,POST,DELETE").
 
+
+extract_username(Form) ->
+    CouchFormat = couch_util:get_value("name", Form),
+    try couch_util:get_value("username", Form, CouchFormat) of
```

Comment:
What's the case for having try/catch here? if `couch_util:get_value` will raise badarg due to `Form`, it'll happen on the line above.